ci/test/reuse

```diff
@@ -51,7 +51,7 @@ action "show env" {
 }
 EOF
 
-popper run --runtime "$RUNTIME" > output
+popper run > output
```

Comment:
looking forward to when we add other runtimes, we could instead add a conditional to check whether `RUNTIME=singularity` is true and skip accordingly